libs/data-mapper-v2/src/images/FunctionIcons.tsx

```diff
@@ -20,9 +20,49 @@ const Count32RegularIcon = (props: IconProps) => {
   );
 };
 
+const CreatePercentageIcon = (props: FluentIconsProps) => {
```

Comment:
Actually, might want to pull the Functions Icon here?

Reply:
I moved it to a separate file. We can keep the icons that represent functions/categories separate.